crypto/ring-signature/src/onetime_keys.rs

```diff
@@ -98,7 +98,7 @@ fn hash_to_scalar(point: RistrettoPoint) -> Scalar {
 /// * `recipient` - The recipient subaddress `(C,D)`.
 pub fn create_tx_out_target_key(
     tx_private_key: &RistrettoPrivate,
-    recipient: &PublicAddress,
+    recipient: &impl RingCtAddress,
```

Comment:
Does this become a breaking change for clients? 

I see it being used in full-service (tests only)

https://github.com/mobilecoinofficial/full-service/blob/f1e58e60e6915710bc45210643dc088dde379a84/full-service/src/test_utils.rs#L459

Reply:
I think it's okay because the current object they are passing implements this trait